File: src/assets/usage-examples-code/windows/set_current_window/set_current_window-1-basic-usage.cpp
#include "splashkit.h"

int main()
{
    window wind1 = open_window("Window 1", 800, 600);
    window wind2 = open_window("Window 2", 800, 600);

    write_line("Setting the current window to 'Window 1'...");
    set_current_window(wind1);
    write_line("Current window is now 'Window 1'.");

    delay(1000); // Delay to keep the window open for 1 second

    write_line("Setting the current window to 'Window 2'...");
    set_current_window(wind2);
    write_line("Current window is now 'Window 2'.");

    delay(3000); // Delay to keep the window open for 3 seconds

    return 0;
}

File: src/assets/usage-examples-code/windows/set_current_window/set_current_window-1-basic-usage.cs
using System;
using SplashKitSDK;

Window wind1 = SplashKit.OpenWindow("Window 1", 800, 600);
Window wind2 = SplashKit.OpenWindow("Window 2", 800, 600);

Console.WriteLine("Setting the current window to 'Window 1'...");
SplashKit.SetCurrentWindow(wind1);
Console.WriteLine("Current window is now 'Window 1'.");

SplashKit.Delay(1000); // Delay to keep the window open for 1 second

Console.WriteLine("Setting the current window to 'Window 2'...");
SplashKit.SetCurrentWindow(wind2);
Console.WriteLine("Current window is now 'Window 2'.");

SplashKit.Delay(3000); // Delay to keep the window open for 3 seconds
